baseplate/core.py

```diff
@@ -187,6 +190,7 @@ def configure_tracing(self, service_name,
                 num_span_workers=num_span_workers,
                 span_batch_interval=span_batch_interval,
                 sample_rate=sample_rate,
+                log_if_unconfigured=log_if_unconfigured,
```

Comment:
Yeah, it would. But now's the time before more services pick it up if we want to do it.